kbcstorage/buckets.py

```diff
@@ -112,12 +113,20 @@ def delete(self, bucket_id, force=False):
             bucket_id (str): The id of the bucket to be deleted.
             force (bool): If ``True``, deletes the bucket even if it is not
                 empty. Default ``False``.
+            asynchronous (bool): if asynchronous then the
```

Comment:
imho introducing the option is unnecessary, because false is going to be deprecated and true will be the only option